tests/BundleInitializationTest.php

```diff
@@ -17,22 +15,34 @@
 use Shivas\VersioningBundle\Twig\VersionExtension;
 use Shivas\VersioningBundle\Writer\VersionWriter;
 use Shivas\VersioningBundle\Writer\WriterInterface;
+use Symfony\Bundle\FrameworkBundle\Test\KernelTestCase;
 
 /**
  * @covers \Shivas\VersioningBundle\ShivasVersioningBundle
  */
-class BundleInitializationTest extends BaseBundleTestCase
+class BundleInitializationTest extends KernelTestCase
 {
     protected function setUp(): void
     {
         parent::setUp();
+    }
 
-        $this->addCompilerPass(new PublicServicePass('/^Shivas\\\\VersioningBundle\\\\|^shivas_versioning.twig.version$/'));
+    protected static function getKernelClass(): string
+    {
+        return \Nyholm\BundleTest\TestKernel::class;
     }
 
-    protected function getBundleClass(): string
+    protected static function createKernel(array $options = []): \Symfony\Component\HttpKernel\KernelInterface
     {
-        return ShivasVersioningBundle::class;
+        /**
```

Comment:
Can be reduced to a single line: `/** @var \Nyholm\BundleTest\TestKernel $kernel */`

Reply:
Done.